CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,219 @@
+
+This Code of Conduct applies to all repositories and communities for Quansight-managed open-source projects regardless of whether or not the repository explicitly includes this document. The Code also applies in public spaces when an individual represents a project or its community. Examples include using an official project e-mail address, posting via an official social media account, or acting as an appointed representative at an online or in-person event. What constitutes representing a project may be further defined and clarified by project maintainers.
+
+Quansight is committed to providing a welcoming and inspiring community for all. Anyone claiming affiliation with Quansight public repositories is required to conform to the Code of Conduct. This Code of Conduct applies to all spaces and public repositories of Quansight on all platforms (including Github). Possible consequences for breaching this Code of Conduct are listed below.
+
+For any queries regarding this Code of Conduct, please contact any member of the Code of Conduct committee. The Code of Conduct committee (for 2021) is:
+
++ Name 1
++ Name 2
++ Name 3
+
+The COC committee consists of (at least) three members of Quansight staff. The term of the membership is one calendar year with the option for renewal if desired (the COC committee may stagger departures to preserve committee memory)
+
+You may alternatively choose to contact one of the Quansight Labs Co-Directors with questions regarding this COC:
+
++ Ralf Gommers (`rgommers@quansight.com`)
++ Tania Allard (`tallard@quansight.com`)
+
+### a. Expected Behavior
+
++ Be empathetic, welcoming, friendly, patient, considerate, and mindful.
+    + Everyone deserves to be treated with kindness and empathy.
++ Remember, you might not be communicating in someone else’s primary language.
+    + If a communication is unclear, ask for clarification respectfully.
++ Offer to help if you see someone struggling or otherwise in need of assistance.
+    + Take care to be kind without being patronizing.
+    + If someone asks you for help, try to be generous with your time.
+    + If you do not have the time, direct them to someone else who might.
++ Be mindful of the words you choose.
+    + Do not insult or put down others.
+    + Take responsibility for your words in all communications (i.e., written or verbal).
++ Be respectful during remote and in-person interactions alike.
++ Adopt habits that are inclusive and productive for community members wherever and whomever they are.
++ Be polite and friendly in all forms of communication.
+    + Remember, with written communication, opportunities for misunderstanding are much greater.
+    + Use sarcasm sparingly.
+    + Make judicious use of emojis to aid in clear online communication (tone is difficult to decipher online).
```

Comment:
Personally I hate it when people make judicious use of emojis, especially in professional settings. Maybe "consider using extratextual indicators such as emojis to ensure that tone is clearly communicated".